r2dbc-mysql/src/main/java/JasyncInsertFakeMetadata.kt

```diff
@@ -0,0 +1,46 @@
+package com.github.jasync.r2dbc.mysql
+
+import io.r2dbc.spi.ColumnMetadata
+import io.r2dbc.spi.RowMetadata
+
+/**
+ * An implementation of [RowMetadata] for support [JasyncStatement.returnGeneratedValues].
+ *
+ * It is also an implementation of [ColumnMetadata] for reduce redundant objects.
+ *
+ * @see JasyncInsertFakeRow
+ */
+internal class JasyncInsertFakeMetadata(private val generatedKeyName: String) : RowMetadata, ColumnMetadata {
```

Comment:
How about calling it `Synthetic` or `GeneratedKeys` instead of `Fake`?

Reply:
Now naming it `JasyncInsertSyntheticMetadata`.